api/v1/models/UniversalTrackingItem.js

```diff
@@ -0,0 +1,17 @@
+var Model = require('./Model');
+
+var UniversalTrackingItem = Model.extend({
+    tableName: 'event_tracking',
+    idAttribute: 'id',
+    hasTimestamps: ['created'],
+    validations: {
```

Comment:
It only persists in the cache for that time period, but it along with related statistics persists in the database for perpetuity